processing/1_create_compendia.py

```diff
@@ -191,6 +191,61 @@
 sample_to_strain_table.head()
 # -
 
+# ## Save pre-binned data with median accessory expression
+# This dataset will be used for Georgia's manuscript, which describes how we generated these compendia
+
+# +
+# Select columns with median accessory expression
+pao1_pa14_acc_expression_select = pao1_pa14_acc_expression[
+    ["median_acc_expression_pao1", "median_acc_expression_pa14"]
+]
+
+pao1_pa14_acc_expression_select.head()
+
+# +
+# Add SRA strain type
+pao1_pa14_acc_expression_label = pao1_pa14_acc_expression_select.merge(
+    sample_to_strain_table, left_index=True, right_index=True
+)
+# Rename column
+pao1_pa14_acc_expression_label = pao1_pa14_acc_expression_label.rename(
+    {"Strain type": "SRA label"}, axis=1
+)
+
+pao1_pa14_acc_expression_label.head()
+
+# +
+# Add our binned label
+pao1_pa14_acc_expression_label["Our label"] = "NA"
+pao1_pa14_acc_expression_label.loc[pao1_binned_ids, "Our label"] = "PAO1-like"
+pao1_pa14_acc_expression_label.loc[pa14_binned_ids, "Our label"] = "PA14-like"
+
+pao1_pa14_acc_expression_label.head()
+
+# +
+# Confirm dimensions
+pao1_expression_prebin_filename = paths.PAO1_PREBIN_COMPENDIUM
+pa14_expression_prebin_filename = paths.PA14_PREBIN_COMPENDIUM
+
+pao1_expression_prebin = pd.read_csv(
```

Comment:
Is the point of splitting up into the two expression prebins just to confirm they're the same size? I think so because you're only saving the combined one. Could you just check sample sizes without having to load in the other files?

Reply:
So the two expression prebins are because the same samples were mapped to 2 different references - a PAO1 and a PA14 reference. So this assertion is to make sure that the number of samples is the same in both, which it is, and that when we added information about our accessory gene expression and labels we retained the same number of samples, which we did. I'll add a comment about this to clarify